src/modules/formatter.py

```diff
@@ -502,7 +502,7 @@ def read_set_to_df(self, test=False):
         # elif not self.emphasis: lines = [re.sub(r',EMPH', '', sentence) for sentence in lines]
 
         if not self.use_semantic_gender:
-            df.message = df.message.str.replace(',(M|F);', ';', regex=True)
+            df.message = df.message.str.replace(',(M|F|M-F);', ';', regex=True)
```

Comment:
This edit is probably redundant, as the corpus generator shouldn't ever produce messages with "M-F" instead of either gender.